2022-07_sprint/schedule.md

```diff
@@ -4,19 +4,17 @@
 :::{include} schedule_table.md
 :::
 
+
 :::{dropdown} Open dropdown for spreadsheet with times in more time zones
 :icon: clock
 
 <iframe src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSJ_x4V63aji-WmarsgEyj9eS_Cp0DB5m647TZdNjYj03nPQq0u0Q6eKPGhHrw-8Ugp16KhAg-AgLkK/pubhtml?widget=true&amp;headers=false"  height="320" width="750"></iframe>
+
```

Comment:
```suggestion
```

Reply:
@OriolAbril 
I tested both on mobile, and the screenshot image+link is easier to navigate.